spec/forms/teacher_interface/add_another_page_form_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe TeacherInterface::AddAnotherPageForm, type: :model do
+  describe "#valid?" do
+    subject(:valid?) { form.valid? }
+
+    let(:form) { described_class.new(add_another:) }
```

Comment:
```suggestion
    subject(:form) { described_class.new(add_another:) }
```

If we change the subject of this block to the form (and remove `:valid?` as the subject), we can do:

```rb
it { is_expected.to_not be_valid }
```

rather than

```rb
it { is_expected.to be false }
```

which I think reads a little nicer.